dockerfiles/QAT/qat.conf

```diff
@@ -0,0 +1,9 @@
+CONFIG_PCIEAER=y
```

Comment:
Sort alphabetically? ;)

Reply:
hmmm. I suspect they are currently in the order they either natively turn up in a kernel config file or that is recommended by some QAT docs somewhere, or 'make sense' in the order they are in (kernel kconfig files are mildly ordered). I'm prone to leave as is unless @mythi or @eadamsintel say differently

Reply:
wait, is this a kernel fragment? why it's not in the fragments directory (like the gpu fragment)?

Reply:
because.... if you read the intent and the code - this gets copied over to the fragment dir during the build. We don't have QAT enabled in kata kernels by default - half the point of this PR is to add a semi-automated way to do a custom Kata rootfs/kernel build ;-)

Reply:
but, I guess I understand maybe what you mean - 'Why do we not just enable this in the default kernel?'. I guess size and stuff. This dockerfile is not different from the current QAT setup we have, which is a markdown doc describing how to do this. We didn't put the QAT kernel fragment in the default kernel at that point either. the current decision I think is to not have all possible hardware enabled in the default kernel. Each separate item I guess requires its own discussion on its merits. As this PR does not change the current status quo, I'll leave it as is, and if somebody wants to debate if QAT should be enabled in the default kata kernel, then raise the question here, or a new Issue, or in the weekly conf call I guess?